### ISSUE TITLE
What to do before the 2nd Session

### ISSUE THREAD

**Issue by supportcs1** [codesydney/summation#11]
1. In GitHub desktop, navigate to File -> New branch to create a new branch. By default, it will link this branch to where you initially cloned the repository (eg. C:/summation).

2. Under 'Current branch', choose your branch name.

****This is where you need to start doing your code changes but at this point just proceed to step 3. We will do code changes eventually.  

3. In GitHub desktop, add a comment to your changes and click the commit button to save the changes. (Thanks for @Jgan91 for letting me know about this). 

4. In Github desktop, click on history and you should see your changes(ie. recent commits).

5. Create pull request via GitHub. This will send a request to the owner to merge your changes to the master.

6. The project owner will confirm that your changes are committed. 

7. Once successfully merged, delete your branch for housekeeping.

**Comment by engramar** [comment]
@Jgan91 @linlingpei @patninad @seyedmohideen @Laurencjusz @supportcs1
Hi Guys,

If you have time, please try to do the above 7 steps before our session next Saturday. Thanks. 

Regards,
Engramar

**Comment by Jgan91** [comment]
@engramar  Is it necessary to do the Git dance if we're using Github Desktop? It looks like we can [commit changes via the GUI]( https://help.github.com/desktop-beta/guides/contributing-to-projects/committing-and-reviewing-changes-to-your-project/).

Is there something I'm missing?

**Comment by engramar** [comment]
@Jgan91 Thanks for letting me know. Looks like you are correct. Let me try and I will let you know. Once confirmed, I may need to tweak the notes. Thanks again.

**Comment by engramar** [comment]
@Jgan91 You are correct mate. Commenting and committing is already in GitHub Desktop. Makes life easier. I have updated the instructions above. Thanks again.
